dashboard/app/models/school.rb

```diff
@@ -16,7 +16,6 @@
 #  address_line3               :string(30)
 #  latitude                    :decimal(8, 6)
 #  longitude                   :decimal(9, 6)
-#  state_school_id             :string(255)
```

Comment:
please do not remove this line yet, it will be removed by the migration (and will cause minor issues if removed sooner)